src/Resources/RoleResource.php

```diff
@@ -167,10 +165,18 @@ public static function table(Table $table): Table
 
     public static function getRelations(): array
     {
-        return [
-            PermissionRelationManager::class,
-            UserRelationManager::class,
-        ];
+
+        $relationManagers = [];
+
+        if (config('filament-spatie-roles-permissions.should_display_relation_managers.permissions')) {
```

Comment:
This should be set to `true` by default.

`if (config('filament-spatie-roles-permissions.should_display_relation_managers.permissions', true))`

Reply:
Good spot. Pushed that up now